cmd/tenant_controller/main.go

```diff
@@ -32,6 +36,18 @@ func init() {
 }
 
 func main() {
+
+	go func() {
+		sigs := make(chan os.Signal, 1)
+		signal.Notify(sigs, syscall.SIGQUIT)
+		buf := make([]byte, 1<<20)
+		for {
+			sig := <-sigs
+			stacklen := runtime.Stack(buf, true)
+			klog.Infof("=== received SIGQUIT (%s) ===\n*** goroutine dump...\n%s\n*** end\n", sig, buf[:stacklen])
+		}
+	}()
+
```

Comment:
The implementation should be integrated into `pkg/signals` and used in _all_ controllers.

Reply:
... relocated accordingly.

Since both controllers (`tenant_controller`, `run_controller`) are using `signals.SetupSignalHandler` this is now used by these/all controllers.